modules/integration/tests-integration/lazy-loading-test/src/test/java/org/wso2/appserver/integration/lazy/loading/util/WebAppStatus.java

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
Seems like you have an identical WebAppStatus.java in tests-common & tests-integration (utils) . Is there any specific reason for duplicating this class ?